website/docs/reference/strategy-variants.mdx

```diff
@@ -1,138 +1,97 @@
 ---
-title: Strategy Variants
+title: Variants
```

Comment:
I think we still have strategy variants and environment variants, so maybe it's worth keeping it (especially because the page is named strategy-variants). I believe @kwasniew may have more context

Reply:
I'll revert this for now, once env. variants are fully removed, we can update the title